index.html

```diff
@@ -23,11 +23,11 @@
                 }
     }
     </style>
-    <script src='http://www.w3.org/Tools/respec/respec-w3c-common' async=""
-    class='remove'>
-</script>
+    <script src='http://www.w3.org/Tools/respec/respec-w3c-common' async class=
+    'remove'>
+    </script>
     <script class='remove'>
-var respecConfig = {
+    var respecConfig = {
```

Comment:
Whoa? What version of tidy are you using? Please use this one: https://github.com/htacg/tidy-html5